CustomOptions.h

```diff
@@ -7,6 +7,11 @@ class CustomOptionsManager
     {
         rightClickDoorOpening.defaultValue = true;
         rightClickDoorOpening.currentValue = true;
+        
+        dismissSound.defaultValue = true;
+        dismissSound.currentValue = true;
+        dismissSound.defaultString = "airLoss";
+        dismissSound.currentString ="airLoss";
     }
 
     static CustomOptionsManager *GetInstance()
```

Comment:
Do we need to bloat this struct? Perhaps we should change this into a templated struct, i.e., Setting<T>

Reply:
Done.